dev-env/bin/ghcide

```diff
@@ -1,4 +1,9 @@
-#!/usr/bin/env bash
-DADE_CURRENT_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-source "$DADE_CURRENT_SCRIPT_DIR/../lib/dade-common"
-bazel build @ghcide-exe//ghcide && bazel-bin/external/ghcide/ghcide-0.7.2.0/_install/bin/ghcide "$@"
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash ../../shell.nix
+
+set -e
```

Comment:
any reason not to go for the standard `set -eou pipefail`?

Reply:
I don't know how that became standard but I don't like it.

1. I find it unclear that the `o` is paired with `pipefail`.
2. We don't use pipes in this script.
3. Something something one thing per line.

I am happy to change this to `set -eu` or `set -e -u` if it makes you happier. :slightly_smiling_face:

Reply:
I get your point but in the end, I find it much easier to enforce that every script starts with the same prelude than make a case by case decision which can easily become outdated later if we start using pipes.
So I think this is one of the case where we’re better off using the same thing everywhere than using each person’s personal taste and having a bunch of different headers. I couldn’t care less about the exact formatting we use for the header as long as its consistent so if you do want to push to change it everywhere, I’m all for it.

Reply:
I defer to the opinion of the master :)
https://cuddly-octo-palm-tree.com/posts/2021-01-31-bash-set-pipefail/

Reply:
That works for me. Done.